src/components/common/TmBalance.vue

```diff
@@ -19,6 +19,24 @@
                 {{ overview.totalStake | bigFigureOrShortDecimals | noBlanks }}
               </h2>
             </div>
+            <div class="currency-selector">
+              <img
+                class="currency-flag"
+                :src="
+                  '/img/icons/currencies/' +
+                    selectedFiatCurrency.toLowerCase() +
+                    '.png'
+                "
+                alt="USA flag"
+              />
+              <select v-model="selectedFiatCurrency">
+                <option>EUR</option>
```

Comment:
should we populate this list dynamically or is hardcoded going to work for now?

Reply:
I think we can leave it hardcoded for now, since we are sending this fixed list of fiatValues from the API